qtwidgetsextra/groupbox/QCheckGroupBoxPlugin.cpp

```diff
@@ -0,0 +1,78 @@
+#include "QCheckGroupBoxPlugin.h"
+#include "QCheckGroupBox.h"
+
+#include <QtPlugin>
+#include <QApplication>
+#include <QStyle>
+
+QCheckGroupBoxPlugin::QCheckGroupBoxPlugin(QObject *parent)
+    : QObject(parent)
+{
+    m_initialized = false;
+}
+
+void QCheckGroupBoxPlugin::initialize(QDesignerFormEditorInterface * /* core */)
+{
+    if (m_initialized) {
+        return;
+    }
+
+    // Add extension registrations, etc. here
+
+    m_initialized = true;
+}
+
+bool QCheckGroupBoxPlugin::isInitialized() const
+{
+    return m_initialized;
+}
+
+QWidget *QCheckGroupBoxPlugin::createWidget(QWidget *parent)
```

Comment:
Just return a bare new widget here, don't add fake items!! The fake items goes into the domXml function.